pkg/apis/flyteworkflow/v1alpha1/workflow.go

```diff
@@ -40,10 +40,13 @@ type FlyteWorkflow struct {
 	NodeDefaults NodeDefaults `json:"node-defaults,omitempty"`
 	// Specifies the time when the workflow has been accepted into the system.
 	AcceptedAt *metav1.Time `json:"acceptedAt,omitempty"`
-	// ServiceAccountName is the name of the ServiceAccount to use to run this pod.
-	// More info: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
-	// +optional
+	// [DEPRECATED] ServiceAccountName is the name of the ServiceAccount to use to run this pod.
+	// [DEPRECATED] More info: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
+	// [DEPRECATED] +optional
 	ServiceAccountName string `json:"serviceAccountName,omitempty" protobuf:"bytes,8,opt,name=serviceAccountName"`
+	// Security context fields to define privilege and access control settings
+	// +optional
+	SecurityContext core.SecurityContext `json:"securityContext,omitempty" protobuf:"bytes,12,rep,name=securityContext"`
```

Comment:
you have to generate code, this will not work.
The deepcopy code has to be generated and to do so is really not easy at the moment

Steps:
1. ensure the propeller code is checked out in `$GOPATH/github.com/flyteorg/flytepropeller`
2. Uncomment https://github.com/flyteorg/flytepropeller/blob/master/hack/tools.go#L5
3. make op_code_generate

Reply:
ohh

Reply:
updated